website/content/en/docs/reference/configuration/_index.md

```diff
@@ -23,6 +23,7 @@ data_dir: "/var/lib/vector"
 
 # Vector's API (disabled by default)
 # Enable and try it out with the `vector top` command
+# NOTE: this is _enabled_ for helm chart deployments. Otherwise it will fail, see: https://github.com/vectordotdev/helm-charts/blob/develop/charts/vector/examples/datadog-values.yaml#L78-L81
```

Comment:
Can you elaborate why this will fail for helm chart deployments?  From my understanding, the API is not required for the helm chart to work.

Reply:
remove "failed" wording